doc/lsst.meas.astrom/tasks/lsst.meas.astrom.matchOptimisticB.MatchOptimisticBTask.rst

```diff
@@ -0,0 +1,99 @@
+.. lsst-task-topic:: lsst.meas.astrom.MatchOptimisticBTask
+
+####################
+MatchOptimisticBTask
+####################
+
+.. Summary paragraph (a few sentences)
+.. The aim is to say what the task is for
+
+``MatchOptimisticBTask`` matches sources to reference objects. This is often done
+as a preliminary step to fitting an astrometric or photometric solution.
+
+Optimistic Pattern Matching is described in V. Tabur 2007, PASA, 24, 189-198
+"Fast Algorithms for Matching CCD Images to a Stellar Catalogue"
+
+.. _lsst.meas.astrom.MatchOptimisticBTask-summary:
+
+Processing summary
+==================
+
+.. If the task does not break work down into multiple steps, don't use a list.
+.. Instead, summarize the computation itself in a paragraph or two.
+
+``MatchOptimisticBTask`` runs this sequence of operations:
+
+- Flags sources with bad centroids and low signal to noise and remove them from
+  the matching.
+- Match the usable sources with an input reference catalog using the V. Tabur
+  2007 algorithm.
+- Further remove sources detected on the edge of the image and those that are
+  saturated.
+- Return these sources matched to the references.
+
+.. _lsst.meas.astrom.MatchOptimisticBTask-api:
+
+Python API summary
+==================
+
+.. lsst-task-api-summary:: lsst.meas.astrom.MatchOptimisticBTask
+
+.. _lsst.meas.astrom.MatchOptimisticBTask-subtasks:
+
+Retargetable subtasks
+=====================
+
+.. lsst-task-config-subtasks:: lsst.meas.astrom.MatchOptimisticBTask
+
+.. _lsst.meas.astrom.MatchOptimisticBTask-configs:
+
+Configuration fields
+====================
+
+.. lsst-task-config-fields:: lsst.meas.astrom.MatchOptimisticBTask
+
+.. _lsst.meas.astrom.MatchOptimisticBTask-examples:
+
+Examples
+========
+
+.. Add a brief example here.
+.. If there are multiple examples
+.. (such as one from a command-line context and another that uses the Python API)
+.. you can separate each example into a different subsection for clarity.
+
+MatchOptimisticBTask is a subtask of AstrometryTask, which is called by
+PhotoCalTask.
```

Comment:
It's whats currently there in MatchOptimisticB and came from the old doxygen. With this I'm more concerned with getting something in there instead of making sure it's completely done.

Reply:
I get that, I just didn't realize it was previously marked as an example.